<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="FX-JSJ800" name="com.zhou.autotest.user.M1_fun_tv_home_user2_Film" tests="4" failures="1" timestamp="22 Apr 2016 05:45:47 GMT" time="66.338" errors="0">
  <testcase name="testClickSkip" time="6.477" classname="com.zhou.autotest.user.M1_fun_tv_home_user2_Film"/>
  <testcase name="testCookie" time="36.204" classname="com.zhou.autotest.user.M1_fun_tv_home_user2_Film"/>
  <testcase name="testLocalStorage" time="3.107" classname="com.zhou.autotest.user.M1_fun_tv_home_user2_Film">
    <failure type="net.sf.json.JSONException" message="null object">
      <![CDATA[net.sf.json.JSONException: null object
	at net.sf.json.JSONObject.verifyIsNull(JSONObject.java:2498)
	at net.sf.json.JSONObject.get(JSONObject.java:1712)
	at com.zhou.autotest.M1_fun_tv_home_common.LocalStorage(M1_fun_tv_home_common.java:268)
	at com.zhou.autotest.user.M1_fun_tv_home_user2_Film.testLocalStorage(M1_fun_tv_home_user2_Film.java:114)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:76)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at com.zhou.run.Run2.main(Run2.java:81)
]]>
    </failure>
  </testcase>
  <testcase name="testFilm" time="20.550" classname="com.zhou.autotest.user.M1_fun_tv_home_user2_Film"/>
</testsuite>
